biz.aQute.launchpad/src/aQute/launchpad/BundleSpecBuilder.java

```diff
@@ -12,6 +12,7 @@
 import org.osgi.framework.BundleActivator;
 import org.osgi.framework.Version;
 
+import aQute.bnd.osgi.Verifier;
```

Comment:
@pkriens May not want this tight linkage to bndlib. He recently removed a reference to Parameters to avoid such a linkage.

Reply:
Thanks for the pointer.

I have changed it to use `Verifier.SYMOBLICNAME_STRING` intead of `Verifier.SYMOBLICNAME`. Because this dependency is a static final String it is resolved at compile time and doesn't create any runtime dependency on bndlib. Hopefully this is sufficient to address @pkriens ' concerns. Otherwise, I'll have to duplicate the string.

Reply:
Please just copy the string, most of the work was make this work without bnd dependencies